csharp/src/ProtocolBuffers.Test/TestProtos/Unittest.cs

```diff
@@ -3303,16 +3323,16 @@ public sealed partial class Builder : pb::GeneratedBuilder<RepeatedGroup, Builde
       if (hasDefaultCord) {
         output.WriteString(85, field_names[2], DefaultCord);
       }
-      if (hasOneofUint32) {
+      if (oneofFieldCase_ == OneofFieldOneofCase.OneofUint32) {
```

Comment:
The Serialization is sorted by field number (after merge fields and extension ranges). That's why I didn't use switch here.